scripts/mupihat/mupihat_bq25792.py

```diff
@@ -1543,47 +5590,49 @@ def read_VBAT_PRESENT(self):
           0h = VBAT NOT present 
           1h = VBAT present
         '''
-        try:
-            #self.read_all_register()
-            # first read register
-            reg_addr = self.REG1D_Charger_Status_2._addr 
-            val = self.bq.read_byte_data(self.i2c_addr, reg_addr)
-            time.sleep(self.busWS_ms/1000)
-            self.registers[reg_addr] = val
-            self.REG1D_Charger_Status_2.set((self.registers[reg_addr]))
-            value, ICO_STAT, TREG_STAT, DPDM_STAT, VBAT_PRESENT_STAT = self.REG1D_Charger_Status_2.get()
-            return VBAT_PRESENT_STAT
-        except Exception as _error:
-            sys.stderr.write('read_VBAT_PRESENT failed, %s\n' % str(_error))
-            if self._exit_on_error: sys.exit(1)
-            value, ICO_STAT, TREG_STAT, DPDM_STAT, VBAT_PRESENT_STAT = self.REG1D_Charger_Status_2.get()
-            return VBAT_PRESENT_STAT
-        finally:
-            pass
+        self.REG1D_Charger_Status_2.get_VBAT_PRESENT_STAT()
```

Comment:
```suggestion
        return self.REG1D_Charger_Status_2.get_VBAT_PRESENT_STAT()
```
Return statement is missing here. Therefore the function always return None.